lib/temporal/client/grpc_client.rb

```diff
@@ -273,16 +273,32 @@ def reset_workflow_execution(namespace:, workflow_id:, run_id:, reason:, workflo
           workflow_execution: Temporal::Api::Common::V1::WorkflowExecution.new(
             workflow_id: workflow_id,
             run_id: run_id,
-            request_id: SecureRandom.uuid
```

Comment:
This isn't a field on `WorkflowExecution` but on the higher `ResetWorkflowExecutionRequest` so I don't think this method would ever have worked.

Reply:
Also it's optional, so providing a random string that we don't return anyway doesn't feel useful.